be/src/exprs/agg/aggregate.h

```diff
@@ -39,13 +39,19 @@ class AggregateFunction {
     virtual void update(FunctionContext* ctx, const Column** columns, AggDataPtr __restrict state,
                         size_t row_num) const = 0;
 
+    // Update the aggregation state with null
+    virtual void update_null(FunctionContext* ctx, AggDataPtr __restrict state) const {}
```

Comment:
Why need the `update_null` and `merge_null` both?  If a aggregate function wants to handle null, I think when update or merge, the process way should be same.

Reply:
Yes, can be combined to one for implement. Splitting into two is for semantic clarity. If combined to one interface,  what should it be named ? update_merge_null() ?

Reply:
handle_null or process_null?

Reply:
In this way, the semantics will not be clear, the semantics are two things, and the implementation of the intermediary is the same, so I feel that it is still necessary to separate.

Reply:
Fixed